package.json

```diff
@@ -51,7 +51,7 @@
     "ts-jest": "^24.0.2",
     "tslint": "^5.11.0",
     "tslint-config-dabapps": "github:dabapps/tslint-config-dabapps#v0.5.3",
-    "typescript": "^3.3.4000"
+    "typescript": "^3.8.2"
```

Comment:
You can't upgrade to `3.4` or above as you'll break all backwards compatibility.

Reply:
Maybe we could release it as a major version? Typescript `3.3.4` couldn't handle making those function generic

Reply:
Or we could not change it. :wink:

Reply:
But then we won't have typed returns for the axios promises :'(

Reply:
What does the typescript version have to do with Axios promises? Axios has supported generic returns for long before TS 3 even.

Reply:
I thought there was something wrong with the types but I might have been mad, I've reverted the version and it works now 🤷‍♂